sky/clouds/gcp.py

```diff
@@ -94,6 +94,11 @@
     f'\nTo query common AI images: {colorama.Style.BRIGHT}gcloud compute images list --project deeplearning-platform-release | less{colorama.Style.RESET_ALL}'
 )
 
+# Image ID tags
+DEFAULT_CPU_IMAGE_ID = 'skypilot:cpu-debian-11'
+DEFAULT_GPU_K80_IMAGE_ID = 'skypilot:k80-debian-10'
+# DEFAULT_GPU_IMAGE_ID = 'skypilot:custom-gpu-debian-11'
+DEFAULT_GPU_IMAGE_ID = 'projects/sky-dev-465/global/images/skypilot-gcp-gpu-debian-11-v20241003'
```

Comment:
Let's use our own tag instead, so that it can be easily updated when we update our hosted catalog.